build/wd_ts_bundle.bzl

```diff
@@ -24,7 +24,9 @@ def wd_ts_bundle(
         internal_wasm_modules = [],
         internal_data_modules = [],
         lint = True,
-        deps = []):
+        deps = [],
+        ts = True,
```

Comment:
what's the idea here? If you don't need typescript use wd_js_bundle, otherwise I don't see apoint.

Reply:
I think the transformations done in  the `wd_js_bundle` invocation here (lines 61 through 108 or so) should be moved into the `wd_js_bundle` command and then we can just pass the arguments here unaltered and I can call `wd_js_bundle` directly.